common/app/routes/Settings/components/DangerZone.jsx

```diff
@@ -58,10 +63,10 @@ class DangerZone extends PureComponent {
             >
             <Alert
               bsStyle='danger'
+              style={centerText}
```

Comment:
Remove this styling, this should be rather handled in a dedicated styles file, for the component group. On that note, for this particular message this is an unneeded change.